### PR TITLE
NLIC-2444: Extend shop token for bundle acquisition

### PR DESCRIPTION
**Pull request by v-rudkovskiy**

### CODE REVIEW

File: NetLicensingClient/src/main/java/com/labs64/netlicensing/domain/Constants.java [hunk 1]
```diff
@@ -58,6 +58,9 @@ public static final class Token {
         public static final String TOKEN_PROP_VENDORNUMBER = "vendorNumber";
         public static final String TOKEN_PROP_SHOP_URL = "shopURL";
         public static final String TOKEN_PROP_PRIVATE_KEY = "privateKey";
+        public static final String TOKEN_PROP_BUNDLE_NUMBER = "bundleNumber";
+        public static final String TOKEN_PROP_BUNDLE_PRICE = "bundlePrice";
+        public static final String TOKEN_PROP_TOKEN_EXPIRATION = "tokenExpiration";
```
Comment (kkorotkov) [inline NetLicensingClient/src/main/java/com/labs64/netlicensing/domain/Constants.java:63]: Is this still valid, considering the changes in PR 220 in the core?

Comment (kkorotkov) [reply NetLicensingClient/src/main/java/com/labs64/netlicensing/domain/Constants.java:63]: I don't see any uses for these three constants either.

Comment (v-rudkovskiy) [reply NetLicensingClient/src/main/java/com/labs64/netlicensing/domain/Constants.java]: I left the constants for users who will be use the library

Comment (kkorotkov) [reply NetLicensingClient/src/main/java/com/labs64/netlicensing/domain/Constants.java]: -> Folllow-up MR adding respective test/exmple.

**Review by kkorotkov — APPROVED** [review]
Approved to move on with the release, however follow-up MR is required to include code exmaple(s) for bundle / token.

**Pull request merged**